src/action.ts

```diff
@@ -384,8 +384,8 @@ class Action {
     let testsTable = "<table><thead><tr><td colspan=5>Tests</td></tr>"
     + "<tr><td>Action</td><td>Passed 🟢</td><td>Skipped ⚪</td><td>Failed 🔴</td><td>Result</></tr></thead><tbody>"
     let vulnerabilitiesTable = "<table><thead><tr><td colspan=8>Vulnerabilities</td></tr>"
-    + "<tr><td>Action</td><td>Minimal</td><td>Low</td><td>Medium</td><td>High</td><td>❗️Critical</td><td>Unknown</td>"
-    + "<td>Result</td></tr></thead><tbody>"
+    + "<tr><td>Action</td><td>Minimal</td><td>Low</td><td>Medium</td><td>High</td>"
+    + "<td>Critical&#x2139</td><td>Unknown</td><td>Result</td></tr></thead><tbody>"
```

Comment:
I added the ℹ️ emoji and removed the❗. I have to split the lines because the linter was complaining due to the length line: `error  This line has a length of 158. Maximum allowed is 140`